samples/snippets/create_kmeans_model_test.py

```diff
@@ -0,0 +1,74 @@
+def test_kmeans_sample():
+    import datetime
+
+    import bigframes
+    from bigframes import dataframe
+    import bigframes.pandas as bpd
+
+    # NOTE:  ask about line below and whether it is needed outside of colab notebooks
```

Comment:
Looks like we shouldn't need to manually set the project. If we decide for clarity to add this, we'll do it across all samples.

Reply:
Removed code that set project manually.